pkg/controller/healthz.go

```diff
@@ -16,15 +18,21 @@ import (
 
 var (
 	rl = rate.NewLimiter(rate.Every(time.Minute), 1)
+
+	mHealthzBuildInfo = stats.Int64("build_info", "Current build information", "1")
 )
 
-func HandleHealthz(hctx context.Context, cfg *database.Config, h *render.Renderer) http.Handler {
-	logger := logging.FromContext(hctx).Named("healthz")
+func HandleHealthz(ctx context.Context, cfg *database.Config, h *render.Renderer) http.Handler {
+	logger := logging.FromContext(ctx).Named("healthz")
 
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		ctx := r.Context()
+		ctx := observability.WithBuildInfo(r.Context())
+
+		// Emit the buildinfo metric
+		stats.Record(ctx, mHealthzBuildInfo.M(1))
```

Comment:
We'll have to treat this metric as normal metric, create a view for it and register with `view.Register()`, even if we only record the metric once for the lifetime of the instance.

The godoc on this: https://godoc.org/go.opencensus.io/stats#example-Record

> // Without any views or exporters registered, this statement has no observable effects.